### DISCUSSION TITLE
Ubuntu - macOS

### DISCUSSION THREAD

**Discussion by TomDev234** [input-leap/input-leap#2138] [Q&A]
How to make Input-Leap work on Ubuntu 24.04 (client) and macOS Sequoia (server)?
I only get error messages.

**Comment by shymega** [comment]
I honestly don't know what you're asking here. You have provided the OS details - that's a start.

But this is like, the bare minimum of the info we need to fix your issue. It doesn't provide any traction for fixing it.

Please provide further details, including, but not limited to:

- DEBUG2 (log level) logs from Input Leap on the client/server.
- Expected behaviour description.
- Current behaviour description.
- Versions of Input Leap.
- Screenshots/videos if possible.

If this is a bug, I will turn it into an issue. But currently this isn't enough to go on _at all_.

**Comment by TomDev234** [comment]
These are the logs are from Debian 12 and macOS Sequoia loglevel Info and Debug Mode 2
same problem errors
no connection established
InputLeap Debian 12 3.0.2 Version: 3.0.2-release-release-7e5889dc
InputLeap macOS Sequoia Version 3.0.2 Version: 3.0.2-release-release-7e5889dc
What screenshots do you need?
[input-leap-client-debug2.log](https://github.com/user-attachments/files/18253259/input-leap-client-debug2.log)
[input-leap-client-info.log](https://github.com/user-attachments/files/18253260/input-leap-client-info.log)
[input-leap-server-debug2_1.log](https://github.com/user-attachments/files/18253261/input-leap-server-debug2_1.log)
[input-leap-server-debug2.log](https://github.com/user-attachments/files/18253262/input-leap-server-debug2.log)
[input-leap-server-info.log](https://github.com/user-attachments/files/18253263/input-leap-server-info.log)

**Comment by TomDev234** [comment]
Connection between Ubuntu 24.04 and Debian 12.08 is working.
Connection between Mac - Linux or Mac - Mac is not working.
There must be a problem with the Mac App.
